### PR TITLE
Changed to module-wise logging

### PR DESCRIPTION
**Pull request by loicbarrault**
Hi, 
I changed the logging to "module-wise" logging so that we can have a better control on logging.
That's important when sacrebleu is used within another tool and you want to deactivate the logging only for it.
Hope it helps.
Cheers, Loic

### CODE REVIEW

File: sacrebleu/sacrebleu.py [hunk 1]
```diff
@@ -149,63 +150,63 @@ def main():
...
+                sacrelogger.error('Unknown test set "%s"', test_set)
+                sacrelogger.error('Please run with --list to see the available test sets.')
                 sys.exit(1)
 
     if args.test_set is None:
         if len(args.refs) == 0:
-            logging.error('I need either a predefined test set (-t) or a list of references')
-            logging.error(get_available_testsets())
+            sacrelogger.error('I need either a predefined test set (-t) or a list of references')
+            sacrelogger.error(get_a_list_of_testset_names())
```
Comment (martinpopel) [inline sacrebleu/sacrebleu.py:183]: Where is `get_a_list_of_testset_names()` defined? How is this edit related to this PR?

File: sacrebleu/sacrebleu.py [hunk 2]
```diff
@@ -220,9 +221,9 @@ def main():
 
     if args.langpair is not None and 'bleu' in args.metrics:
         if args.langpair.split('-')[1] == 'zh' and args.tokenize != 'zh':
-            logging.warning('You should also pass "--tok zh" when scoring Chinese...')
+            logger.warning('You should also pass "--tok zh" when scoring Chinese...')
```
Comment (martinpopel) [inline sacrebleu/sacrebleu.py:224]: Why `logger` instead of `sacrelogger`?

File: sacrebleu/utils.py [hunk 3]
```diff
@@ -193,12 +194,12 @@ def download_test_set(test_set, langpair=None):
...
         with portalocker.Lock(lockfile, 'w', timeout=60):
             if not os.path.exists(tarball) or os.path.getsize(tarball) == 0:
-                logging.info("Downloading %s to %s", dataset, tarball)
+                sacrelogger.info("Downloading %s to %s", dataset, tarball)
                 try:
                     with urllib.request.urlopen(dataset) as f, open(tarball, 'wb') as out:
                         out.write(f.read())
                 except ssl.SSLError:
-                    logging.warning('An SSL error was encountered in downloading the files. If you\'re on a Mac, '
+                    sacrelogger.addHandlerarning('An SSL error was encountered in downloading the files. If you\'re on a Mac, '
```
Comment (martinpopel) [inline sacrebleu/utils.py:202]: `addHandlerarning`???

**Comment by loicbarrault** [comment]
Hi guys, sorry for this PR full of rubbish... I rushed to do it (before going in holidays) and didn't check carefully. I will solve the bugs before resubmitting after next week.

File: sacrebleu/sacrebleu.py (hunk 1 above)
Comment (loicbarrault) [reply sacrebleu/sacrebleu.py:183]: Thanks Martin, I merged from a previous version with vimdiff, and it seems I did a wrong copy/paste here. Sorry for that.

File: sacrebleu/sacrebleu.py (hunk 2 above)
Comment (loicbarrault) [reply sacrebleu/sacrebleu.py:224]: Of course! Thanks Martin!

File: sacrebleu/sacrebleu.py (hunk 1 above)
Comment (loicbarrault) [reply sacrebleu/sacrebleu.py]: Now fixed!

File: sacrebleu/sacrebleu.py (hunk 2 above)
Comment (loicbarrault) [reply sacrebleu/sacrebleu.py]: Now fixed!

File: sacrebleu/utils.py (hunk 3 above)
Comment (loicbarrault) [reply sacrebleu/utils.py]: Now fixed. Sorry again for this...

**Review by martinpopel — APPROVED** [review]
